doc/api/child_process.md

```diff
@@ -842,7 +842,7 @@ pipes between the parent and child. The value is one of the following:
    file, socket, or a pipe with the child process. The stream's underlying
    file descriptor is duplicated in the child process to the fd that
    corresponds to the index in the `stdio` array. The stream must have an
-   underlying descriptor (file streams do not until the `'open'` event has
+   underlying descriptor (file streams do not open until the `'open'` event has
```

Comment:
I'm not sure if "open" is the correct term here, maybe "start"? @nodejs/streams @nodejs/child_process wdyt?

Reply:
I'm not a native speaker but "start" seems better. However, I think "open" is fine too.